models/meshmodel/registry/registry.go

```diff
@@ -440,6 +465,22 @@ func (rm *RegistryManager) GetRegistrant(e Entity) Host {
 	_ = rm.db.Where("id = ?", reg.RegistrantID).Find(&h).Error
 	return h
 }
+func (rm *RegistryManager) GetRegisteriesCount() (v1alpha1.EntitySummary, error) {
```

Comment:
Please rename this function. There is only one Registry. There may be multiple Registrants (hosts that register Models, which contain Components).